workshops/hosting-workshop/README.md

```diff
@@ -0,0 +1,119 @@
+---
+name: 'Deploying Websites'
+description: 'Deploying, hosting, using continuous deployment, custom domains and more'
+author: '@aaryanporwal'
+img: 'https://cloud-kv2ikdnkl.vercel.app/0image.png'
+---
+
+# Deploying, Hosting, Continuous Deployment and Custom Domains
+
+In this workshop we'll be learning how to push your website on GitHub (A website that provides hosting for software development and version control using Git), hosting it on [Netlify](https://www.netlify.com) with a custom domain from [namecheap](https://www.namecheap.com).
+
+## Part 1: The Prerequisites
+
+1. A GitHub Account
```

Comment:
this list looks a bit awkward imo, one line is super long the other is super short

Reply:
Thanks for this review, and please keep suggesting more :)